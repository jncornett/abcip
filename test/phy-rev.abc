a(syn)
b(syn, ack)
a(ack)

a(ack, len=64, phy:rev=3)
a(ack, len=64)
a(ack, len=64)
b(ack)

a(fin,ack)
b(fin,ack)
a(ack)

