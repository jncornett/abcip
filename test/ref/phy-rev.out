IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto TCP (6), length 40)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [S], cksum 0xb3d9 (correct), seq 1, win 8192, length 0
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0028 0001 0000 4006 5cbc 0a01 0203 0a09  .(....@.\.......
	0x0020:  0807 bdec 0008 0000 0001 0000 0000 5002  ..............P.
	0x0030:  2000 b3d9 0000                           ......
IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto TCP (6), length 40)
    10.9.8.7.8 > 10.1.2.3.48620: Flags [S.], cksum 0xb3c7 (correct), seq 1, ack 2, win 8192, length 0
	0x0000:  0201 0203 0405 0209 0807 0605 0800 4500  ..............E.
	0x0010:  0028 0001 0000 4006 5cbc 0a09 0807 0a01  .(....@.\.......
	0x0020:  0203 0008 bdec 0000 0001 0000 0002 5012  ..............P.
	0x0030:  2000 b3c7 0000                           ......
IP (tos 0x0, ttl 64, id 2, offset 0, flags [none], proto TCP (6), length 40)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [.], cksum 0xb3c8 (correct), ack 1, win 8192, length 0
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0028 0002 0000 4006 5cbb 0a01 0203 0a09  .(....@.\.......
	0x0020:  0807 bdec 0008 0000 0002 0000 0002 5010  ..............P.
	0x0030:  2000 b3c8 0000                           ......
IP (tos 0x0, ttl 64, id 5, offset 0, flags [none], proto TCP (6), length 104)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [.], cksum 0xedc3 (correct), seq 129:193, ack 1, win 8192, length 64
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0068 0005 0000 4006 5c78 0a01 0203 0a09  .h....@.\x......
	0x0020:  0807 bdec 0008 0000 0082 0000 0002 5010  ..............P.
	0x0030:  2000 edc3 0000 692c 2061 2069 6d70 6572  ......i,.a.imper
	0x0040:  6469 6574 206c 6163 7573 2065 6e69 6d20  diet.lacus.enim.
	0x0050:  6120 6665 6c69 732e 2043 756d 2073 6f63  a.felis..Cum.soc
	0x0060:  6969 7320 6e61 746f 7175 6520 7065 6e61  iis.natoque.pena
	0x0070:  7469 6275 7320                           tibus.
IP (tos 0x0, ttl 64, id 4, offset 0, flags [none], proto TCP (6), length 104)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [.], cksum 0x039f (correct), seq 65:129, ack 1, win 8192, length 64
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0068 0004 0000 4006 5c79 0a01 0203 0a09  .h....@.\y......
	0x0020:  0807 bdec 0008 0000 0042 0000 0002 5010  .........B....P.
	0x0030:  2000 039f 0000 7573 6369 7069 742c 2071  ......uscipit,.q
	0x0040:  7561 6d20 6174 2073 6365 6c65 7269 7371  uam.at.scelerisq
	0x0050:  7565 2075 6c74 7269 6369 6573 2c20 6e69  ue.ultricies,.ni
	0x0060:  7369 206e 756c 6c61 2066 6572 6d65 6e74  si.nulla.ferment
	0x0070:  756d 206f 7263                           um.orc
IP (tos 0x0, ttl 64, id 3, offset 0, flags [none], proto TCP (6), length 104)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [.], cksum 0x701b (correct), seq 1:65, ack 1, win 8192, length 64
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0068 0003 0000 4006 5c7a 0a01 0203 0a09  .h....@.\z......
	0x0020:  0807 bdec 0008 0000 0002 0000 0002 5010  ..............P.
	0x0030:  2000 701b 0000 4c6f 7265 6d20 6970 7375  ..p...Lorem.ipsu
	0x0040:  6d20 646f 6c6f 7220 7369 7420 616d 6574  m.dolor.sit.amet
	0x0050:  2c20 636f 6e73 6563 7465 7475 7220 6164  ,.consectetur.ad
	0x0060:  6970 6973 6369 6e67 2065 6c69 742e 2045  ipiscing.elit..E
	0x0070:  7469 616d 2073                           tiam.s
IP (tos 0x0, ttl 64, id 2, offset 0, flags [none], proto TCP (6), length 40)
    10.9.8.7.8 > 10.1.2.3.48620: Flags [.], cksum 0xb308 (correct), ack 193, win 8192, length 0
	0x0000:  0201 0203 0405 0209 0807 0605 0800 4500  ..............E.
	0x0010:  0028 0002 0000 4006 5cbb 0a09 0807 0a01  .(....@.\.......
	0x0020:  0203 0008 bdec 0000 0002 0000 00c2 5010  ..............P.
	0x0030:  2000 b308 0000                           ......
IP (tos 0x0, ttl 64, id 6, offset 0, flags [none], proto TCP (6), length 40)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [F.], cksum 0xb307 (correct), seq 193, ack 1, win 8192, length 0
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0028 0006 0000 4006 5cb7 0a01 0203 0a09  .(....@.\.......
	0x0020:  0807 bdec 0008 0000 00c2 0000 0002 5011  ..............P.
	0x0030:  2000 b307 0000                           ......
IP (tos 0x0, ttl 64, id 3, offset 0, flags [none], proto TCP (6), length 40)
    10.9.8.7.8 > 10.1.2.3.48620: Flags [F.], cksum 0xb306 (correct), seq 1, ack 194, win 8192, length 0
	0x0000:  0201 0203 0405 0209 0807 0605 0800 4500  ..............E.
	0x0010:  0028 0003 0000 4006 5cba 0a09 0807 0a01  .(....@.\.......
	0x0020:  0203 0008 bdec 0000 0002 0000 00c3 5011  ..............P.
	0x0030:  2000 b306 0000                           ......
IP (tos 0x0, ttl 64, id 7, offset 0, flags [none], proto TCP (6), length 40)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [.], cksum 0xb306 (correct), ack 2, win 8192, length 0
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0028 0007 0000 4006 5cb6 0a01 0203 0a09  .(....@.\.......
	0x0020:  0807 bdec 0008 0000 00c3 0000 0003 5010  ..............P.
	0x0030:  2000 b306 0000                           ......
